### ISSUE TITLE
property for "Amount" of results

### ISSUE THREAD

**Issue by ttodua** [mattzollinhofer/vue-typeahead-bootstrap#88]
I've been reading through docs and couldn't find...
Where is the property, which says how many items are matching the current query?
i.e. `MatchedAmount` or whatever, which will provide that i.e. 4  (or whatever items were found in dropdown for the entered query).

Found method `matchedItems` in https://github.com/mattzollinhofer/vue-typeahead-bootstrap/blob/master/src/components/VueTypeaheadBootstrap.vue , in which method might `MatchedAmount`  can be set,   but i am not sure how to create a new property `MatchedAmount`  exposed to outside world.

**Comment by mattzollinhofer** [comment]
Are you looking for `maxMatches`?

**Comment by ttodua** [comment]
No, I think i have not explained. I dont want the input argument to define (maxMatches is just argument for VTB), I mean, when you search for any query and in dropdown it finds X matched amount. I want that X number. So, how many matches are being showing in dropdown (if i have maxMatches 1000, but it gets i.e. 4 results for query, that result is what I want to know).

**Comment by mattzollinhofer** [comment]
oh oh, I see. I can see that being helpful and I don't think that should be too hard to do. I need to look through issues and see what needs to be done next, but I'll definitely put this on the list.
